digitaltwins/samples/device/javascript/sample_device.js

```diff
@@ -31,15 +32,20 @@ const propertyUpdateHandler = async (component, propertyName, reportedValue, des
 
 const commandHandler = (request, response) => {
   console.log('received command: ' + request.commandName + ' for component: ' + request.componentName);
+  // blink
+  // turn on
+  // turnoff
+  // run diag
   response.acknowledge(200, 'helpful response text');
   console.log('acknowledgement succeeded.');
 };
 
 
 const environmentalSensor = new EnvironmentalSensor('sensor', propertyUpdateHandler, commandHandler);
 const deviceInformation = new DeviceInformation('deviceInformation');
+const sdkInformation = new SDKInformation('SDKInformation');
 
-const modelId = 'dtmi:YOUR_COMPANY_NAME_HERE:sample_device;1';
+const modelId = 'dtmi:my_company:com:sample_device;1';
```

Comment:
>my_company [](start = 22, length = 10)

assuming that this looks the same as "c".

Reply:
Yes sir.